pkg/nf90io/NF90IO.h

```diff
@@ -0,0 +1,41 @@
+C $Header: /u/gcmpack/MITgcm/pkg/kl10/KL10.h,v 1.1 2014/07/30 03:28:05 jmc Exp $
+C $Name: checkpoint66d $
+
+#ifdef ALLOW_NF90IO
+
+CBOP
+C !ROUTINE: NF90IO.h
+
+C !DESCRIPTION: \bv
+C     /==========================================================\
+C     | NF90IO.h                                                   |
+C     | o Basic header for Netcdf/Fortran90/Parallel i/o           |
+C     \==========================================================/
+
+C-----------------------------------------------------------------------
+C
+C Constants that can be set in data.nf90io
```

Comment:
If there is not going to be a `data.nf90io` file, this comment will need changing.